docs/INSTALL.md

```diff
@@ -74,28 +74,41 @@ command line scripts as explained in the [scripts README](https://github.com/Ada
     ```
     /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
     ```
-3) Install python3 and libsodium
+3) Install automake, libtool, and libsodium
     ```
-    brew install python libsodium
+    brew install automake libtool libsodium
     ```
-4) Create virtualenv "jmvenv"
-    ```sh
-    pip3 install virtualenv
-    virtualenv jmvenv
-    source jmvenv/bin/activate
+4) Build secp256k1
+    ```
+    git clone https://github.com/bitcoin-core/secp256k1
+    cd secp256k1
+    git reset --hard 0d9540b13ffcd7cd44cc361b8744b93d88aa76ba
```

Comment:
Why `git reset --hard` instead of `git checkout`?

Reply:
No strong opinion here... Do you think one is better than the other?

Reply:
"checkout" is the proper way, "Switch branches or restore working tree files" vs "Reset current HEAD to the specified state".